test/es6/generators-functionality.js

```diff
@@ -1378,7 +1378,7 @@ var tests = [
             var iteratorReturningWithoutValue = CreateIterable(() => { return {done: false}; }, () => { return {done: true}; }, () => { return {done: true}; });
             gf = function* () { yield* iteratorReturningWithoutValue; }
             g = gf();
-            assert.areEqual({value: undefined, done: false}, g.next(), "As the value property is missing undefined is returned");
+            g.next();

```

Comment:
Because it tested the wrong behavior fixed by this pr. I removed the assertion because the correct behavior is tested by the test I added.